keyboards/keebio/bdn9/keymaps/eosti/config.h

```diff
@@ -1 +1,17 @@
+/* Copyright 2020 Reid Sox-Harris
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
```

Comment:
```suggestion
#pragma once

```

Reply:
In a different PR, I was told that I should have a license in both the keymap.c and config.h files by drashana. Which is preferred?

Reply:
Both the licence header, and the header guard

Reply:
Got it, thanks!